<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>700</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="pages">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>700</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>480</width>
      <height>700</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>480</width>
      <height>700</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="start_page">
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>700</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>480</width>
       <height>700</height>
      </size>
     </property>
     <widget class="QPushButton" name="start_sign_in_button">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>450</y>
        <width>400</width>
        <height>40</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Sign In</string>
      </property>
     </widget>
     <widget class="QLabel" name="start_welcome_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>400</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Welcome to Crazy Suite&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="start_register_button">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>510</y>
        <width>400</width>
        <height>40</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sign_in_page">
     <widget class="QPushButton" name="back_to_start_from_sign_in">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="sign_in_username_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>315</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sign_in_username_text">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>400</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="sign_in_password_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>405</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sign_in_password_text">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>400</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sign_in_sign_in_button">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>500</y>
        <width>400</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Sign In</string>
      </property>
     </widget>
     <widget class="QLabel" name="register_label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>400</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Sign In&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="register_page">
     <widget class="QLineEdit" name="register_password_text">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>435</y>
        <width>400</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="register_username_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="register_submit_button">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>585</y>
        <width>400</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QLabel" name="register_password_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>400</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_to_start_from_register">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="register_username_text">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>345</y>
        <width>400</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="register_password_confirm_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>490</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password Confirm:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="register_password_confirm_text">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>525</y>
        <width>400</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="register_email_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="register_email_text">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>255</y>
        <width>400</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="register_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>95</y>
        <width>400</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Register&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="products">
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
